src/uwtools/config.py

```diff
@@ -80,6 +80,35 @@ def dump_file(self, output_path):
         ''' Interface to write a config object to a file at the
         output_path provided. '''
 
+    @abc.abstractmethod
```

Comment:
I don't _think_ this needs to be an abstract method since it is fully implemented here. 

If you can remove that, you should also be able to remove the additions of the methods to the sub-classes. Those should only need implementing if they somehow differ from this implementation.

Reply:
Hmm, I'll take a look - I debated that, but was keeping the existing behavior since we don't really call the base method outside of here. Should be able to make it a class method which will be a good starting before we add the args, thanks!

Reply:
Class method applied without issue